portfolio/src/main/webapp/script.js

```diff
@@ -54,3 +54,17 @@ function updateButtonsState(visibleImgId) {
     nextButton.disabled = false;
   }
 }
+
+function toggleTheme() {
+  var toggler = document.getElementById('theme-toggler');
+  var body = document.getElementsByTagName('body')[0];
```

Comment:
Same as in other functions, it's a good practice to use `const` by default, unless when you actually when to mutate the variable, then change to `var`.